src/ReleaseHistory.md

```diff
@@ -1,7 +1,9 @@
 # SARIF Package Release History (SDK, Driver, Converters, and Multitool)
+
 ## **v2.3.7** [Sdk](https://www.nuget.org/packages/Sarif.Sdk/2.3.7) | [Driver](https://www.nuget.org/packages/Sarif.Driver/2.3.7) | [Converters](https://www.nuget.org/packages/Sarif.Converters/2.3.7) | [Multitool](https://www.nuget.org/packages/Sarif.Multitool/2.3.7) | [Multitool Library](https://www.nuget.org/packages/Sarif.Multitool.Library/2.3.7)
 * DEPENDENCY BREAKING: SARIF now requires Newtonsoft.JSON 11.0.2 (rather than 10.0.3)
 * DEPENDENCY: SARIF TypeScript package now requires minimist 1.2.3 or later (rather than >=1.2.0)
+* FEATURE: Multitool SARIF output is now pretty-printed by default. To remove white space, specify `--minify`. [#2098](https://github.com/microsoft/sarif-sdk/issues/2098)
```

Comment:
>FEATURE [](start = 2, length = 7)

These changes don't appear to merge cleanly with 'main', can you address that if true?

Reply:
Merged, pushed.

---
In reply to: [500529805](https://github.com/microsoft/sarif-sdk/pull/2100#discussion_r500529805) [](ancestors = 500529805)